lib/ui/painting.dart

```diff
@@ -4283,6 +4283,175 @@ class Canvas extends NativeFieldWrapperClass2 {
 
   /// Draws the given [Image] into the canvas with its top-left corner at the
   /// given [Offset]. The image is composited into the canvas using the given [Paint].
+  ///
+  /// An example usage to draw an asset:
```

Comment:
you'll want to use the `{@tool` stuff to mark up sample code. cc @gspencergoog is that documented anywhere?

Reply:
Yes. https://github.com/flutter/flutter/blob/master/dev/snippets/README.md